modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElixirClientCodegen.java

```diff
@@ -872,6 +896,9 @@ private boolean getRequiresHttpcWorkaround() {
 
     class ExtendedCodegenModel extends CodegenModel {
         public boolean hasImports;
+        public List<CodegenProperty> primitiveVars = new ArrayList<>();
```

Comment:
These are not strictly required, but they allow us to keep the model template simpler.